t/14-grutasks.t

```diff
@@ -72,13 +73,14 @@ $assets_result_mock->redefine(
 $assets_result_mock->redefine(refresh_size => sub { });
 
 # setup test config and database
-my $test_case     = OpenQA::Test::Case->new(config_directory => "$FindBin::Bin/data/41-audit-log");
-my $schema        = $test_case->init_data(fixtures_glob => '01-jobs.pl 04-products.pl');
-my $jobs          = $schema->resultset('Jobs');
-my $job_groups    = $schema->resultset('JobGroups');
-my $parent_groups = $schema->resultset('JobGroupParents');
-my $assets        = $schema->resultset('Assets');
-my $job_assets    = $schema->resultset('JobsAssets');
+my $test_case        = OpenQA::Test::Case->new(config_directory => "$FindBin::Bin/data/41-audit-log");
+my $schema           = $test_case->init_data(fixtures_glob => '01-jobs.pl 04-products.pl');
+my $jobs             = $schema->resultset('Jobs');
+my $job_groups       = $schema->resultset('JobGroups');
+my $job_dependencies = $schema->resultset('JobDependencies');
```

Comment:
I've just been adding this line to the block. The other lines are only changed by tidy.